Remotion/Mixins/Core/Definitions/AttributeDefinition.cs

```diff
@@ -61,7 +61,7 @@ public void Accept (IDefinitionVisitor visitor)
       visitor.Visit (this);
     }
 
-    public string FullName
+    public string? FullName
```

Comment:
I could add another assertion resulting in 

```csharp
public string FullName
{
  get
  {
    var declaringType = _data.Constructor.DeclaringType;
    Assertion.IsNotNull (declaringType);
    return Assertion.IsNotNull (declaringType.FullName);
  }
}
```

Reply:
that's TODO https://re-motion.atlassian.net/browse/RM-7814 + https://re-motion.atlassian.net/browse/RM-7811, so we can leave it for now :)

Reply:
Alright, but should the signature be changed now?

Reply:
as in, make it nullable for now and then change it back? since we're doing the same in other places, too, I would say, yes, we change it now instead of investing in cleanup up-front. 

from a pure git-history PoV, clenaup this up first and then not needing to change the signature would be the preferred approach, but this would add quite a bit to the current PR, or rather, be a separate PR first and then some merge conficts and in total more than a couple of minutes of work. So, yeah, we keep it nullable here until we do the big fullname/declaringtype cleanup. Not perfect, but probably "better" from a planning PoV. I am flexible, though, if you want to do the cleanup first :)

Reply:
Makes sense.